src/guides/v2.3/javascript-dev-guide/widgets/widget_calendar.md

```diff
@@ -41,6 +41,28 @@ require([
 });
 ```
 
+## Validate the calendar widget
+
+To validate the calendar widget, required **dateFormat** parameter should be specified, as can be seen from the validate-date rule initialisation:
```

Comment:
@EvheniiaKolkhiieva  could you please highlight a validate-date - `validate-date`.

Thank you!